<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI Light</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:none;</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>20</y>
     <width>108</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_Little">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: none;
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::ListRemove"/>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Large">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
	border: none;
	color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::MediaPlaybackStop"/>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Close">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: none;
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #A0A0A0;
border-radius: 10px;
padding: 5px; 
selection-background-color: lightblue;
selection-color: black;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_member_1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 213, 251, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
background-color: transparent; 
border: none;
}


QGroupBox QPushButton:hover{
border: 2px;
border-radius: 10px;
background-color:rgb(255, 102, 105);
color:white;
padding-left: 10px;

}
QPushButton{
border: 2px;
background-color: transparent; 
border-radius: 10px;
}

</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI Light</family>
         <pointsize>11</pointsize>
         <kerning>true</kerning>
         <fontweight>Light</fontweight>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>推荐</string>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushButton_1_0.png</normaloff>:/icon/img/pushButton_1_0.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>精选</string>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushButton_2_0.png</normaloff>:/icon/img/pushButton_2_0.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft YaHei UI Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>动态</string>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushButton_3_0.png</normaloff>:/icon/img/pushButton_3_0.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{
background-color: transparent; 
border: none;
}


QGroupBox QPushButton:hover{
border: 2px;
border-radius: 10px;
background-color:rgb(255, 102, 105);
color:white;
padding-left: 10px;

}
QPushButton{
border: 2px;
background-color: transparent; 
border-radius: 10px;
}

</string>
    </property>
    <property name="title">
     <string>我的</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>我喜欢的音乐</string>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushButton_4_0.png</normaloff>:/icon/img/pushButton_4_0.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>最近播放</string>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushButton_5_0.png</normaloff>:/icon/img/pushButton_5_0.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI Light</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>下载的音乐</string>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushButton_6_0.png</normaloff>:/icon/img/pushButton_6_0.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>951</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">margin:0px;
padding:0px;
</string>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QWidget" name="widget_MusicPlayer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>610</y>
     <width>1151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color:rgb(255, 255, 255);
}</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>1</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_player_left">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushbutton_player_left.png</normaloff>:/icon/img/pushbutton_player_left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PlayerAction">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushButton_PlayerAction_0.png</normaloff>:/icon/img/pushButton_PlayerAction_0.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_player_right">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="matter.qrc">
         <normaloff>:/icon/img/pushbutton_player_right.png</normaloff>:/icon/img/pushbutton_player_right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_player_like">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="matter.qrc">
      <normaloff>:/icon/img/pushButton_player_like_1.png</normaloff>:/icon/img/pushButton_player_like_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_player">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>331</width>
      <height>5</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_all_time">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>53</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_music_player_img_music">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	border:1px solid rgb(102, 102, 102);
}</string>
    </property>
    <property name="text">
     <string>img</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_music_player_music_name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>music_name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_music_player_music_composer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>music_composer</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>19</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="matter.qrc">
     <normaloff>:/icon/img/pushButton_search.png</normaloff>:/icon/img/pushButton_search.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="matter.qrc"/>
 </resources>
 <connections/>
</ui>
